Source/DafnyRuntime/DafnyRuntimeRust/src/lib.rs

```diff
@@ -216,7 +216,7 @@ pub mod dafny_runtime_conversions {
         }
     }
 
-    // --unicode-chars:false
+    // --unicode-char:false
     pub mod unicode_chars_false {
```

Comment:
I wish I could change the module name here but it's a breaking change, so I'm not sure it's be a good idea at this point.